src/redux/reducers/userReducer.ts

```diff
@@ -22,6 +22,10 @@ export const userReducer = (
         ...emptyUser,
       };
     case SET_USER_SUCCESS:
+      const affiliations = action.user['custom:affiliation']
+        .replace(/[[\]]/g, '')
+        .split(',')
+        .map(affiliationString => affiliationString.trim()) as Affiliation[];
```

Comment:
dette ser sikkert litt rart ut, men det er bare fordi det er rart.
incognito returnerer ett attributt for affiliations som er `"[member, employee, staff]"` og ikke `"[\"member\", \"employee\", \"staff\"]"` som vi måtte hatt for å kunne parse direkte til JSON array. Kunne fikset dette med en liten kodesnutt på backend også, men tenker det ikke er der skoen trykker for tiden så vi kan bare ta det selv

Reply:
ja, ser weird ut, men enig i at det ikke haster. men skriv det ned en plass, så vi husker å si ifra om det!

Reply:
kan lage en sak med lav prio i jira